components/match2/wikis/counterstrike/match_group_input_custom.lua

```diff
@@ -0,0 +1,692 @@
+---
+-- @Liquipedia
+-- wiki=counterstrike
+-- page=Module:MatchGroup/Input/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Json = require('Module:Json')
+local Logic = require('Module:Logic')
+local MathUtil = require('Module:MathUtil')
+local Lua = require('Module:Lua')
+local Opponent = require('Module:Opponent')
+local Table = require('Module:Table')
+local TypeUtil = require('Module:TypeUtil')
+local Variables = require('Module:Variables')
+local Streams = require('Module:Links/Stream')
+local EarningsOf = require('Module:Earnings of')
+
+local MatchGroupInput = Lua.import('Module:MatchGroup/Input', {requireDevIfEnabled = true})
+
+local ALLOWED_STATUSES = { 'W', 'FF', 'DQ', 'L', 'D' }
+local ALLOWED_VETOES = { 'decider', 'pick', 'ban', 'defaultban' }
+local NP_MATCH_STATUS = {'cancelled','canceled', 'postponed'}
+local MAX_NUM_OPPONENTS = 2
+local MAX_NUM_PLAYERS = 10
+local MAX_NUM_MAPS = 9
+local DUMMY_MAP_NAME = 'null' -- Is set in Template:Map when |map= is empty.
+
+local FEATURED_TIERS = { 'S-Tier', 'A-Tier'}
```

Comment:
```suggestion
local FEATURED_TIERS = { 'S-Tier', 'A-Tier'}
```
either have the space between the striings and the table brackets or not
but not a mix of it^^